app/controllers/provider_foreman_controller.rb

```diff
@@ -368,27 +341,23 @@ def tree_select_unprovisioned_configured_system
   def configuration_manager_providers_tree_rec
     nodes = x_node.split('-')
     case nodes.first
-    when "root" then rec = find_record(ManageIQ::Providers::ConfigurationManager, params[:id])
+    when "root" then find_record(ManageIQ::Providers::ConfigurationManager, params[:id])
```

Comment:
from the question before. Does `find_record` buy us anything here? Can we just use a simple find?

Having said that, I'm ok with "those changes are out of scope for this PR."